package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "middy-middleware-jwt-auth",
-  "version": "3.0.4",
+  "version": "4.0.0",
```

Comment:
Version number and changelog are managed by semantic release and will be added automatically once the PR is merged. Please remove these changes from the PR.
For this to work, the commit message is important: the commit message text will be used for the change log. Please add the following at the end of the commit message to ensure that the major version is bumped correctly and the changelog has an explanation:
```
BREAKING CHANGE: As of this version this middleware only supports middy@1.0.0 and upwards. See https://github.com/middyjs/middy/blob/master/UPGRADE.md for middy upgrade instructions.
```